aws/fedora-coreos/kubernetes/nlb.tf

```diff
@@ -17,9 +17,9 @@ resource "aws_route53_record" "apiserver" {
 resource "aws_lb" "nlb" {
   name               = "${var.cluster_name}-nlb"
   load_balancer_type = "network"
-  internal           = true
+  internal           = false
```

Comment:
This and the `subnets` should remain the same (internal/private)

Reply:
@bendrucker This was not even shown as a merge conflict. So strange.